src/net/index.ts

```diff
@@ -52,14 +42,23 @@ export default class Net {
     }
   }
 
+  // Remove a transaction from the stored transaction list
+  public forgetTransaction(hash: string): void {
+    const index = this.transactions.map((t) => t.hash).indexOf(hash);
+
+    if (index > -1) {
+      this.transactions.splice(index, 1);
+    }
+  }
+
   // Empty transactions list
   public emptyTransactions(): void {
     this.transactions = [];
   }
 
   // returns number of pending (confirming) transactions
   public pendingTransactions(): number {
-    return this.transactions.filter((t) => t.status === 'confirming').length;
+    return this.transactions.filter((t) => t.confirmations < 12).length;
```

Comment:
can we make the number of confirmations be set in config as a constant?

Reply:
Updated.